qermit/__init__.py

```diff
@@ -20,14 +20,9 @@
 modify the expectation value of some observable (MitEx).
 """
 from qermit.taskgraph.task_graph import TaskGraph  # noqa:F401
-from qermit.taskgraph.mittask import (  # noqa:F401
-    MitTask,
-    AnsatzCircuit,
-    CircuitShots,
-    ObservableExperiment,
-)
+from qermit.taskgraph.mittask import MitTask, AnsatzCircuit, CircuitShots, ObservableExperiment  # noqa:F401
 from qermit.taskgraph.mitres import MitRes  # noqa:F401
 from qermit.taskgraph.mitex import MitEx  # noqa:F401
-from qermit.taskgraph.utils import SymbolsDict, MeasurementCircuit, ObservableTracker  # noqa:F401
+from qermit.taskgraph.utils import SymbolsDict, MeasurementCircuit, ObservableTracker # noqa:F401
```

Comment:
Not meaning to get too into the weeds here, but this seems to contradict with PEP8. In particular "E261: at least two spaces before inline comment". I prefer to stick to PEP8 as I understand that it is the 'standard', although this could also be a version problem. Is there a way to make black be consistent here (see the imports above with two spaces for example)

Reply:
Don't you worry, this was picked up by the CI & also was a user issue and not a black issue.

Black converted the import to 
```
from qermit.taskgraph.utils import (
 SymbolsDict,
 MeasurementCircuit```
 etc... which was completely at odds with PEP8 so I manually rejigged it.